cmd/simulator/load/loader.go

```diff
@@ -129,5 +138,12 @@ func ExecuteLoader(ctx context.Context, config config.Config) error {
 		return err
 	}
 	log.Info("Tx agents completed successfully.")
+
+	if config.ExposeMetrics {
+		// Start a prometheus server to expose individual tx metrics
+		http.Handle("/metrics", promhttp.HandlerFor(reg, promhttp.HandlerOpts{Registry: reg}))
+		log.Info("localhost:8082/metrics")
```

Comment:
can we provide a more informative message here and make this port a constant

Reply:
done